reactor-core/src/main/java/reactor/core/publisher/EmitterProcessor.java

```diff
@@ -51,7 +51,8 @@
  * @param <T> the input and output value type
  *
  * @author Stephane Maldini
- * @deprecated Prefer clear cut usage of {@link Sinks}, to be removed in 3.5
+ * @deprecated To be removed in 3.5. Prefer clear cut usage of {@link Sinks}:
```

Comment:
Consider also mentioning that `EmitterProcessor` was/is blocking on `emitNext`, while with the new API one can use `tryEmitNext` + `while` with `parkNanos` or similar waiting strategy to achieve the same